math/papilo/Portfile

```diff
@@ -15,12 +18,30 @@ maintainers             {@catap korins.ky:kirill} openmaintainer
 description             Parallel Presolve for Integer and Linear Optimization
 long_description        {*}${description}
 
-checksums               rmd160  086c74c8ce81f8b10c7073ea3b0090a7d1e77e0f \
-                        sha256  f443e94de1ea61132526a68a681e8aa2e94488802085115284cf5dd000b31d25 \
-                        size    1082818
+checksums               rmd160  12630b841611066934262475e29c475051a0c390 \
+                        sha256  a90c93fb680d83e1190ce3930dd67a5dc68f5e983e18f4239963d11bc9de7625 \
+                        size    1083482
 
+compilers.setup         require_fortran
 compiler.cxx_standard   2014
 
 compiler.blacklist-append {clang < 1000}
 
-depends_lib-append      port:onetbb
+depends_lib-append      port:gmp \
+                        port:OpenBLAS \
```

Comment:
I *think* typically the openblas dependency is specified such that the -devel port could also satisfy the dependency. Is that possible here as well (you can grep the ports to see how it's typically done).

Reply:
Thanks. I’ve missed that a few months ago